pkg/standalone/standalone.go

```diff
@@ -931,3 +975,37 @@ func checkAndOverWriteFile(filePath string, b []byte) error {
 	}
 	return nil
 }
+
+func getDefaultRegistry() string {
+	imageRegistryName := githubContainerRegistryName
+	if val := os.Getenv("DEFAULT_IMAGE_REGISTRY"); val != "" {
```

Comment:
Actually this is not needed ... 
image-registry flag is part of viper bindings .... already looks for env `DAPR_IMAGE_REGISTRY` and populates when viper.Get() is called ....

Reply:
@mukundansundar  If you are referring to `viper.BindPFlag("image-registry", cmd.Flags().Lookup("image-registry"))` , then the above is different.

It meant to set the mode to switch to github/docker. May be I need to rename the Env variable to more meaning name? Any suggestions?

Reply:
Why switch unless the image-registry is given? Or is it more only between GHCR and Dockerhub ... ?

Reply:
by default registry will be set to GHCR, if someone wants to change to dockerub then he can do by setting env variable.